static/js/pages/CourseSearchPage.js

```diff
@@ -232,7 +232,16 @@ export default function CourseSearchPage(props: Props) {
   const resultsColumnWidth = searchResultLayout === SEARCH_GRID_UI ? 9 : 8
   const suggestions =
     !emptyOrNil(suggest) && !emptyOrNil(text)
-      ? R.without([text], suggest).map(
+      ? R.without(
+        [
+          text
+            .toLowerCase()
+            .replace(/^"(.*)"$/, "$1")
+            .replace(/[\W]+/g, " ")
+            .trim()
```

Comment:
Can we move this before the double quote regex so we can handle ` "search phrase"` for example?

Reply:
Thanks, looks like I need it in both places (for example, ` "aerospace: biomedical!" `